app/flatpak-builtins-build-init.c

```diff
@@ -186,6 +189,9 @@ flatpak_builtin_build_init (int argc, char **argv, GCancellable *cancellable, GE
   gboolean is_app = FALSE;
   gboolean is_extension = FALSE;
   gboolean is_runtime = FALSE;
+  g_autoptr(GKeyFile) keyfile = g_key_file_new ();
```

Comment:
Small point: Can avoid an alloc/free here in the error case if this initialization is moved closer to where the keyfile is used.

Reply:
I don't think optimizing for the error case is really meaningful.